docs/artifactory/builddockercreate/help.go

```diff
@@ -1,11 +1,9 @@
 package builddockercreate
 
-const Description = "Build Docker Create."
+const Description = "Add a published docker image to the build-info."
 
-var Usage = []string{"jfrog rt build-docker-create <target repo> <--image-file=*file-path*>"}
+var Usage = []string{"jfrog rt build-docker-create <target repo> --image-file=<file-path>"}
```

Comment:
```suggestion
var Usage = []string{"jfrog rt build-docker-create <target repo> --image-file=<Image file path>"}
```
(See jfrog rt u --help for reference)